Ginger/Ginger/Repository/RepositoryItemFactory.cs

```diff
@@ -744,9 +744,22 @@ public void DownloadSolution(string SolutionFolder)
             {
                 sol.ExistInLocaly = true;
             }
-            else if (WorkSpace.Instance.UserProfile.SourceControlType == SourceControlBase.eSourceControlType.GIT && Directory.Exists(PathHelper.GetLongPath(sol.LocalFolder + @"\.git")))
+            else if (WorkSpace.Instance.UserProfile.SourceControlType == SourceControlBase.eSourceControlType.GIT)
             {
```

Comment:
1- need to check which actual folder we try to pull the root folder of solution or the inner ".git" folder
2- test it for gitLab + gitHub solutions